src/templates/static/legal/tac_sections/tac.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const Tac = () => (
     <div>
         <h2 data-anchor='terms-and-conditions'>{it.L('Terms and conditions')}</h2>
-        <p><i>{it.L('Version 53. Last modified 10-02-2022.')}</i></p>
+        <p><i>{it.L('Version 53. Last modified 01-03-2022.')}</i></p>
```

Comment:
the date was changed due to requirements of other task and slack discussion.

https://redmine.deriv.cloud/issues/52818#Update_PA_T&Cs_(Binary)
https://deriv-group.slack.com/archives/C02E0CSRBUJ/p1644302935386899

Reply:
You can remove the external links as our codebase is open source!